ingestion/src/metadata/ingestion/source/database/datalake.py

```diff
@@ -164,9 +202,13 @@ def yield_database_schema(
         )
 
     def _list_s3_objects(self, **kwargs) -> Iterable:
-        paginator = self.client.get_paginator("list_objects_v2")
-        for page in paginator.paginate(**kwargs):
-            yield from page["Contents"]
+        try:
+            paginator = self.client.get_paginator("list_objects_v2")
+            for page in paginator.paginate(**kwargs):
+                yield from page["Contents"]
+        except Exception as exc:
+            logger.debug(traceback.format_exc())
+            logger.warning(f"Unexpected exception to yield s3 object [{page}]: {exc}")
```

Comment:
Just out of curiosity, do you know what error we are trying to catch?

Reply:
one possible error here is that we do not have enough access to list `list_objects_v2`